File: client-app/src/app/model/ListingAggregate/Objects/Owner.ts
import { CompanyAddress, CompanyContacts, CompanyContent, CompanyDescription, RedressScheme } from './../../CompanyAggregate/Company';
import { Insurance } from '../../CompanyAggregate/Insurance';
import { ServiceCategory } from '../../ServiceCategory';

export interface Owner {
    id: string;
    addedOn: Date;
    companyAddress: CompanyAddress;
    companyContacts: CompanyContacts;
    companyContents: CompanyContent;
    companyDescriptions: CompanyDescription;
    LegalName: string;
    companyReference: string;
    displayName: string;
    insurances: Insurance[];
    redressSchemes: RedressScheme[];
    serviceLocations: string;
    summaryDescription: string;
    serviceCategories: ServiceCategory[];
    listingsCount: number;
}

File: Application/ListingApplication/ListingDtos/Owner.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Domain.Enums;
using Domain.CompanyAggregate;
using Domain.ListingAggregate;
using Domain.LocationAggregate;
using Domain;
using Application.CompanyApplication.CompanyDtos;

namespace Application.ListingApplication.ListingDtos
{
    public class Owner
    {
        public Guid Id { get; set; }
        public DateTime AddedOn { get; set; }
        public CompanyAddressDto CompanyAddress { get; set; }
        public CompanyContactsDto CompanyContacts { get; set; }
        public string LegalName { get; set; }
        public string CompanyReference { get; set; }
        public string DisplayName { get; set; }
        public List<RedressScheme> RedressSchemes { get; set; } 
        public string SummaryDescription { get; set; }

    }
}